frb_codegen/src/commands.rs

```diff
@@ -95,7 +104,7 @@ fn cbindgen(
         no_includes: true,
         // copied from: dart-sdk/dart_api.h
         // used to convert Dart_Handle to Object.
-        after_includes: Some("typedef struct _Dart_Handle* Dart_Handle;".to_owned()),
+        after_includes: Some(declarations),
```

Comment:
nit: move the comment of "copied from ... to Object." to where the `typedef struct _DartHandle` lives

Reply:
Refined in [744984a](https://github.com/fzyzcjy/flutter_rust_bridge/pull/1145/commits/744984aac3d8d9ad13f9b0de0723c58a4f59f024).